src/forklift/core.py

```diff
@@ -281,12 +273,23 @@ def _hash(crate, hash_path, needs_reproject):
     insert_cursor = arcpy.da.InsertCursor(changes.table, changes.fields)
 
     with arcpy.da.SearchCursor(crate.source, fields, sql_clause=sql_clause) as cursor:
+        def parse_id_as_int(id):
+            return str(int(id))
+
+        def parse_id_as_string(id):
+            return str(id)
+
+        if crate.source_primary_key_type == int:
```

Comment:
ok it is confusing that if it's an int your casting it to int again. We might add a comment to this to make that more clear.